pkg/trace/api/api.go

```diff
@@ -140,6 +141,7 @@ func (r *HTTPReceiver) Listen(addr, logExtra string) error {
 	r.server = &http.Server{
 		ReadTimeout:  timeout,
 		WriteTimeout: timeout,
+		ErrorLog:     fmtlog.New(logWriter(log.Error), "http.Server: ", 0),
```

Comment:
Will this use the agent logger with its throttling mechanism?

Reply:
I'd say so, given that it forwards all calls to `log.Error`, which uses the underlying logger set up with throttling.

Reply:
Right got a bit mixed up with the `fmtlog` vs `log` packages :+1:

Reply:
No wonder. I've changed the names of that in #3134. All seelog stuff will be `seelog.*`